rdrf/rdrf/forms/widgets/widgets.py

```diff
@@ -322,22 +322,16 @@ def _get_value_range(self, cde_name):
 
 
 class RadioSelect(widgets.RadioSelect):
-    # def __init__(self, name, value, attrs, renderer):
-    #     super(RadioSelect, self).__init__(renderer=renderer)
+    template_name = "rdrf_cdes/radio_select.html"
 
     @staticmethod
     def usable_for_types():
         return {CommonDataElement.DATA_TYPE_RANGE}
 
-    def render(self, name, value, attrs=None, renderer=None):
-        html = super().render(name, value, attrs, renderer)
-        return self._transform(html)
-
-    def _transform(self, html):
```

Comment:
Many other CDEs use similar hacky methods to render the HTML, have a look at the rest of this file. Do you want me to look into fixing those using a similar approach to this one?

Reply:
You can create a JIRA task for it, but let's not do large refactorings until we haven't done the merge work for Angelman yet.